src/pug/blog/report_22.pug

```diff
@@ -0,0 +1,49 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #22
+
+block content
+  h2 Progress report #22
+  +news_header('12 April 2024', 'report_21', '')
+
+  h3 Wireless Alpakka (the odyssey continues)
+  p In the last report we explained our criteria to evaluate #[em Pico-w + BT] (with and without dongle) and how we were not able to achieve acceptable results.
+
+  p During the last weeks we focused on evaluating #[em Pico-w + WLAN], and while the initial results on latency (~2 ms) and jitter were very satisfactory, we found critical issues with #[+ax('https://github.com/georgerobotics/cyw43-driver/issues/111') connection dropping] that we were no able to solve either, which also makes the solution unsuitable.
+
+  p Ultimately we have to ask ourselves if BT or WLAN are the #[em right tool] for low-latency videogame peripherals, and the answer is possibly "no".
+
+  p So even though we may go back to one of these solutions if the blockers are solved, for now we decided to explore alternatives without the Pico-W:
+
+  ul
+    li Keep the original Pico (or bare #[em RP2040]), but with a #[em standalone RF module] best suited for our usecase.
+    li Redesign the PCB into the #[em Nordic Semi] SoC ecosystem.
+    li Redesign the PCB into the #[em Espressif ESP32] SoC ecosystem.
+
+
+  p In the case of Nordic or ESP32 redesigns, we would also have to retain retrocompatibility for current PCBs, which is not a trivial task, so we are exploring how it could be done with #[+ax('https://www.zephyrproject.org') Zephyr] since it provides multiple architectures as a target.
+
+  p We are also tempted to create our own #[em core module] similar in size to a Raspberry Pico, including SoC, wireless module, battery charging, and USB-C. So it plugs into the main PCB with a #[em board-to-board connector], and can be reused in all our devices (Alpakka, Kapybara, dongle, and other device ideas we have for the future).
+
+  hr
+  h3 Battery charging
+  p We have been experimenting with #[em Microchip MCP73871] integrated circuit for power management, as a simple way to power the Alpakka either from the USB or from a LiPo battery, to charge the battery if necessary, and to indicate if the battery is #[em low], #[em charging], or #[em fully charged]. Results are very positive so we will continue with this chip if there are no surprises.
+
+  p With an early prototype we ran a full cycle of charging the battery, and discharging it while playing (mostly Helldivers 2 and other FPS games) and the #[em 1200 mAh] battery lasted for a total of #[em 17 hours] of continued gameplay, which is awesome.
+
+  hr
+  h3 Wireless ready when?
```

Comment:
Remove this section.
It's manipulative (people will go straight here because they want to know the answer).
You do not actually provide an answer.
Instead you beg people to buy.